### ISSUE TITLE
Deploying throws std::filesystem::filesystem_error

### ISSUE THREAD

**Issue by Felolie** [CHollingworth/Lampray#141]
Hey,
maybe it's already solved, but I didn't find exactly my problem.  If I deploy the Mods, it crashes in the end. At the first deploy I get an invalid machine code and at the second time that form above 
"terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
  what():  filesystem error: cannot rename: Device or resource busy [/home/clas/.steam/steam/steamapps/common/Baldurs Gate 3] [/home/clas/.steam/steam/steamapps/common/Lampray Managed - Baldurs Gate 3]"

[Log_Lampray.txt](https://github.com/CHollingworth/Lampray/files/14806513/Log_Lampray.txt)

Maybe some mods are as well fatal.
![Bildschirmfoto vom 2024-03-29 17-31-47](https://github.com/CHollingworth/Lampray/assets/165409323/05494cb3-bd44-40c6-94af-4e7ff5b333a7)

Some Data of my System: 
openSUSE Tumbleweed - Gnome 46
Wayland
Kernel Linux 6.8.1-1-default

Thanks in advance

**Comment by xhBIROhx** [comment]
Same issue here
```
terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
  what():  filesystem error: cannot rename: Invalid cross-device link [/home/xhbirohx/.local/share/Steam/steamapps/common/Baldurs Gate 3/] [/home/xhbirohx/.local/share/Steam/steamapps/common/Baldurs Gate 3/Lampray Managed - ]
fish: Job 1, 'sudo ./Lampray' terminated by signal SIGABRT (Abort)
```

**Comment by simns** [comment]
Any update on this? I just did a fresh install from source of Lampray and I'm getting this same error except mine is
```
filesystem error: cannot rename: Invalid argument
```
I'm only trying this with a single mod, _Transmog Enhanced_.

I'm also on OpenSUSE TW.

**Comment by Felolie** [comment]
No I didn't try it further after a short while. But now you can activate mods in the game and don't need the extrinsic work (but didn't test it long just 1-2 sessions).

**Comment by Felolie** [comment]
Didn't want to close it, sry 😅

**Comment by simns** [comment]
That's true, but there are plenty of mods not available in the in-game mod manager. I feel like I can't quite be satified without the extra mods. :smile: 

Perhaps we can reopen this? It's certainly an open bug. :sweat_smile:
